### DISCUSSION TITLE
Unable to load code as extension

### DISCUSSION THREAD

**Discussion by lucasfth** [PhilipFlyvholm/learnit-plus-plus#45] [Q&A] [answered]
After the addition to run `npm run dev` I have been unable to see the changes I make.
Before I could run `npm run build` and then add the `dist` folder as the unpacked extension in Chrome.
I tried both with `npm run dev` and `npm run build`, but it does not load the extension correctly.

Do you have an idea of what the problem is?

**Comment by PhilipFlyvholm** [comment, marked as answer]
This is because of the new framework we have started using. It would be a good idea to maybe run `npm run i` again if you haven't done this, just to be sure. Then after running `npm run dev` a dev build will show up in the `build` folder called something like `chrome-mv3-dev`. If you point Chrome to this folder instead then it should update. Running `npm run build` will create a production build in `build/chrome-mv3-prod`.

**Reply by lucasfth** [reply]
Thanks <3
